src/code_7bb60_len_41b0.c

```diff
@@ -270,7 +482,42 @@ s32 check_input_hammer(void) {
 
 INCLUDE_ASM(s32, "code_7bb60_len_41b0", check_input_jump);
 
+// control flow, temps, blah
+#ifdef NON_MATCHING
+void check_input_spin(void) {
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    Temp8010F250* temp_8010F250 = &D_8010F250;
+
+    if (!(playerStatus->flags & 0x5000) &&
+        !(playerStatus->animFlags & 1) &&
+        !(playerStatus->currentButtons & D_CBUTTONS) &&
+        !is_ability_active(Ability_SLOW_GO)) {
+
+        s8 actionState = playerStatus->actionState;
+
+        if ((playerStatus->pressedButtons & Z_TRIG) &&
+            (actionState != 0x21) &&
+            (actionState < 0x22) &&
+            (actionState < 3) &&
+            (actionState >= 0) &&
+            !(playerStatus->animFlags & 0x10000)) {
+
+            set_action_state(ActionState_SPIN);
+            if (temp_8010F250->unk_01 != 0) {
+                if (temp_8010F250->unk_08 == 0) {
+                    if (temp_8010F250->unk_0C == 0) {
```

Comment:
does this not match with `&&`?

Reply:
There's other stuff that's wrong, so I'd rather fix the function and then do that